filters/filters.go

```diff
@@ -38,10 +38,13 @@ type FilterContext interface {
 	// route.
 	Served() bool
```

Comment:
Well, this is just the interface. The proxy package provides its own filterContext implementation and it can share the state in any other way, available only to the package.

I'll deprecate this method and figure out a way for the proxy's internal filterContext